projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -106,6 +106,7 @@ export class AppExtensionService implements RuleContext {
   profile: ProfileState;
   repository: RepositoryInfo;
   withCredentials: boolean;
+  aosPlugin: boolean;
```

Comment:
not sure if this is the best place where to add the plugin logic. Usually, every plugin has its own service i.e. MicrosoftExtensionService or ProcessServiceExtensionService

Reply:
An example [here](https://github.com/Alfresco/alfresco-apps/blob/66e0fe66adeeff9f8e6f80a15cc6af7d0f5b4158/libs/content-ee/microsoft-office-online-integration-extension/src/lib/services/microsoft-extension.service.ts#L70)